docker/compose/docker-compose-services.yml

```diff
@@ -47,3 +47,13 @@ kafka:
   volumes:
     - ${DOCKER_DATA_HOME}/kafka:/tmp/kafka-logs
     - ${DOCKER_DATA_HOME}/zookeeper:/var/lib/zookeeper
+
+riakcs:
+  extends:
+    file: docker-compose-base.yml
+    service: riakcs
+  ports:
+    - "9880:8080"
```

Comment:
Use port 9880 instead of 8080 on docker host because port 8080 is often used by other test servers.